dashboard/app/assets/stylesheets/application.scss

```diff
@@ -452,6 +452,23 @@ img.video_thumbnail {
   }
 }
 
+#curriculum-catalog-container {
+  #header-banner {
+    background-color: $darkest_gray;
+  }
+
+  .contentContainer {
+    display: flex;
+    justify-content: center;
+    margin-top: 16px;
+
+    .content {
+      width: 100%;
+      margin: 0 16px;
+    }
+  }
+}
```

Comment:
please do not add new styles to this file. please use react to apply the style directly like you are doing for `styles.content`, or use a .module.scss file [like this](https://github.com/code-dot-org/code-dot-org/blob/f21f811a109e871c6f4200862c04ca23f709d2cb/apps/src/templates/teacherDashboard/PrintCertificates.jsx#L6).